paddle/framework/op_registry.h

```diff
@@ -74,7 +74,8 @@ class OpRegistry {
                                                 const VariableNameMap& outputs,
                                                 AttributeMap attrs);
 
-  static std::unique_ptr<OperatorBase> CreateOp(const OpDesc& op_desc);
+  static std::unique_ptr<OperatorBase> CreateOp(const OpDesc& op_desc,
```

Comment:
What is the difference between the two signatures of CreateOp -- with and without ProgramDesc?

Reply:
There are three `CreateOp` here.
1. One is to look up and invoke Op's constructor.
2. The second is to use protobuf message `ProgramDesc` to create operator.
3. The third is to use a Python binding object `ProgramDescBind` to create operator

The second and third `CreateOp` invoke the first `CreateOp` inside.